fe/fe-core/src/main/java/org/apache/doris/persist/gson/GsonUtils.java

```diff
@@ -978,4 +986,26 @@ public T read(JsonReader reader) throws IOException {
         }
     }
 
+    public static void toJsonCompressed(DataOutput out, Object src) throws IOException {
+        ByteArrayOutputStream byteStream = new ByteArrayOutputStream();
+        try (GZIPOutputStream gzipStream = new GZIPOutputStream(byteStream)) {
+            try (OutputStreamWriter writer = new OutputStreamWriter(gzipStream)) {
+                GsonUtils.GSON.toJson(src, writer);
+            }
+        }
+        Text text = new Text(byteStream.toByteArray());
```

Comment:
![image](https://github.com/user-attachments/assets/2a3ff74f-405d-479f-a82d-a1e8ac412b88)
Is it correct to use `Text`?

Reply:
`Text` also supports writing bytes directly without encoding.